api/src/main/java/io/strimzi/api/kafka/model/kafka/KafkaClusterSpec.java

```diff
@@ -139,8 +139,8 @@ public void setRack(Rack rack) {
         this.rack = rack;
     }
 
-    @Description("Storage configuration (disk). Cannot be updated.")
-    @JsonProperty(required = true)
+    @Description("Storage configuration (disk). Cannot be updated. " +
+            "This property is required when Node Pools are not used.")
```

Comment:
```suggestion
            "This property is required when node pools are not used to set the configuration.")
```

Reply:
This suggestion seems wrong. When the node pools are used, the storage has to be configured there. It is not optional.

Reply:
(I applied the rest of the change)